components/lcm/org.wso2.carbon.apimgt.lifecycle.manager.core/pom.xml

```diff
@@ -51,4 +54,25 @@
         </plugins>
     </build>
 
+    <dependencies>
+        <dependency>
+            <groupId>org.wso2.carbon</groupId>
+            <artifactId>org.wso2.carbon.logging</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.wso2.carbon</groupId>
+            <artifactId>org.wso2.carbon.utils</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.ws.commons.axiom.wso2</groupId>
+            <artifactId>axiom</artifactId>
+            <version>1.2.11.wso2v10</version>
```

Comment:
We should not add dependency version in child module's pom. Version should be inherited from parent pom. Please check other dependencies.